vortex-scalar/src/scalarvalue/mod.rs

```diff
@@ -92,15 +92,16 @@ impl Display for InnerScalarValue {
                 }
             }
             Self::BufferString(bufstr) => {
-                if bufstr.len() > 10 {
-                    write!(
-                        f,
-                        "{}..{}",
-                        &bufstr[0..5],
-                        &bufstr[bufstr.len() - 5..bufstr.len()],
-                    )
+                let bufstr = bufstr.as_str();
```

Comment:
I think it would be nice if a scalar display will effect the display of its scalar value